sui/src/benchmark/load_generator.rs

```diff
@@ -204,3 +204,268 @@ pub async fn spawn_authority_server(
 pub fn calculate_throughput(num_items: usize, elapsed_time_us: u128) -> f64 {
     1_000_000.0 * num_items as f64 / elapsed_time_us as f64
 }
+
+async fn send_tx_chunks_for_quorum_notif(
+    notif: Arc<Notify>,
+    tx_chunk: Vec<Bytes>,
+    result_chann_tx: &mut MpscSender<(u128, usize)>,
+    net_client: NetworkClient,
+    stake: usize,
+    conn: usize,
+) {
+    notif.notified().await;
+    let r = send_tx_chunks(tx_chunk, net_client.clone(), conn).await;
+
+    match result_chann_tx.send((r.0, stake)).await {
+        Ok(_) => (),
+        Err(e) => {
+            // Disconnect is okay since we may leave f running
+            if !e.is_disconnected() {
+                panic!("Send failed! {:?}", net_client)
+            }
+        }
+    }
+
+    let _: Vec<_> =
+        r.1.par_iter()
+            .map(|q| check_transaction_response(deserialize_message(&(q.as_ref().unwrap())[..])))
```

Comment:
Should we make things fail or print errors is there are errors here? (althrough I think the `check_transaction_response` does that?)

Reply:
`check_transaction_response` already does that